### ISSUE TITLE
Create a reporter for app usage stats

### ISSUE THREAD

**Issue by limikael** [tunapanda/swag-old#3]
[ARBTT](http://arbtt.nomeata.de/#how) looks promising for this.

For details on what a reporter is, see [here](http://discourse.tunapanda.org/t/swag-reporters-explained/).

**Comment by limikael** [comment]
Copy of an old issue, see discussion here:

https://github.com/tunapanda/old.swag/issues/9

**Comment by jasonmule** [comment]
Wrote a script that uses the Python Tincan API and ARBTT csv reports: https://github.com/jasonmule/arbtt-swag-reporter

**Comment by limikael** [comment]
This is awesome!

**Comment by jasonmule** [comment]
So, you should be ablet to use a free Scorm cloud account (https://cloud.scorm.com/) for testing.

**Comment by limikael** [comment]
That's quite clever for testing!

(Would have saved me a bit of time compared to installing learninglocker
1000 times if I would have thought of that...)

**Comment by jasonmule** [comment]
Yeah. They provide you with a sandbox and you should be able to view your stored statements when you go to the LRS realm.

**Comment by jasonmule** [comment]
A note for testers: On your Scorm account, key represents your username and secret represents your password. Your endpoint will look like https://cloud.scorm.com/tc/GUU54A3ORL/sandbox/ if you are posting to your sandbox. You should be good to go after you have put these in your config.
